cli/src/main/java/com/michelin/ns4kafka/cli/services/ResourceService.java

```diff
@@ -71,14 +73,25 @@ public Resource getSingleResourceWithType(ApiResource apiResource, String namesp
 
     public HttpResponse<Resource> apply(ApiResource apiResource, String namespace, Resource resource, boolean dryRun) {
         try {
+            // If the resource is a subject, load the schema
+            if (resource.getKind().equals("Schema")) {
+                fileService.loadSchemaFileContent(resource);
```

Comment:
loadSchema only if ``schemaFile`` is there. Remember 
````yaml
schema: {"type": "string"}
# or
schema: |
    {
      "type": "string"
    }
# or
schemaFile: schema.avsc
````

Reply:
Fixed